### ISSUE TITLE
`sfss` Error: missing field bucket, line: 5, column: 1

### ISSUE THREAD

**Issue by brian6932** [winpax/sfsu#41]
Same issue as #12, but this time for `sfss` instead

<details>
  <summary>panic</summary>

```rust
❯ sfss node
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `bucket`", line: 5, column: 1)', D:\a\sfsu\sfsu\src\packages.rs:78:74
stack backtrace:
   0:     0x7ff682871f42 - git_patch_owner
   1:     0x7ff6828895eb - git_odb_object_size
   2:     0x7ff68286e41a - git_patch_owner
   3:     0x7ff682871c8b - git_patch_owner
   4:     0x7ff6828740f9 - git_patch_owner
   5:     0x7ff682873d7a - git_patch_owner
   6:     0x7ff682874981 - git_patch_owner
   7:     0x7ff68287470d - git_patch_owner
   8:     0x7ff68287296f - git_patch_owner
   9:     0x7ff6828743d0 - git_patch_owner
  10:     0x7ff68297ffb5 - git_smart_subtransport_git
  11:     0x7ff6829802c3 - git_smart_subtransport_git
  12:     0x7ff682797e4c - <unknown>
  13:     0x7ff68278d220 - <unknown>
  14:     0x7ff6827825e8 - <unknown>
  15:     0x7ff68278d3e3 - <unknown>
  16:     0x7ff682790133 - <unknown>
  17:     0x7ff68297e3ba - git_smart_subtransport_git
  18:     0x7ff682782748 - <unknown>
  19:     0x7ff68278d3e3 - <unknown>
  20:     0x7ff6827826e8 - <unknown>
  21:     0x7ff68278d3e3 - <unknown>
  22:     0x7ff6827825e8 - <unknown>
  23:     0x7ff68278d3e3 - <unknown>
  24:     0x7ff6827825e8 - <unknown>
  25:     0x7ff68278d3e3 - <unknown>
  26:     0x7ff682790133 - <unknown>
  27:     0x7ff68297e3ba - git_smart_subtransport_git
  28:     0x7ff682782748 - <unknown>
  29:     0x7ff68278d3e3 - <unknown>
  30:     0x7ff682790133 - <unknown>
  31:     0x7ff68297e3ba - git_smart_subtransport_git
  32:     0x7ff682782748 - <unknown>
  33:     0x7ff68278d3e3 - <unknown>
  34:     0x7ff6827826e8 - <unknown>
  35:     0x7ff68278d3e3 - <unknown>
  36:     0x7ff6827825e8 - <unknown>
  37:     0x7ff68278d3e3 - <unknown>
  38:     0x7ff682790133 - <unknown>
  39:     0x7ff68297e3ba - git_smart_subtransport_git
  40:     0x7ff682782114 - <unknown>
  41:     0x7ff68278db7a - <unknown>
  42:     0x7ff682791162 - <unknown>
  43:     0x7ff68297e3ba - git_smart_subtransport_git
  44:     0x7ff6828620e3 - git_patch_owner
  45:     0x7ff68285f292 - git_patch_owner
  46:     0x7ff68285f8f1 - git_patch_owner
  47:     0x7ff682878e4c - git_filter_source_repo
  48:     0x7ffa51ed117e - BaseThreadInitThunk
  49:     0x7ffa533042db - RtlUserThreadStart
  ```

</details>

<details>
  <summary>scoop list</summary>

```pwsh
❯ scoop list | ? { $_.Name -eq 'nodejs' }
Installed apps:

Name   Version Source           Updated             Info
----   ------- ------           -------             ----
nodejs 19.0.1  <auto-generated> 2022-11-21 16:24:30 Held package
```

</details>

<details>
  <summary>sfsl (with hooks)</summary>

```pwsh
❯ scoop list | ? { $_.Name -eq 'nodejs' }

Name   Version Source  Updated
----   ------- ------  -------
nodejs 19.0.1  Invalid 11/21/2022 9:24:15 PM
```

</details>

**Comment by jewlexx** [comment]
Could you attach which version of `sfsu` are you using and a list of your buckets?

**Comment by brian6932** [comment]
`1.1.5`

**Comment by jewlexx** [comment]
Should be fixed in #47, if you are able to test that would be great, because I am unable to re-create the problem on my machine.
